src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4552,7 +4552,7 @@ private static void ReportDuplicateObjectMemberInitializers(BoundExpression boun
             InitializerExpressionSyntax initializerSyntax,
             TypeSymbol initializerType,
             DiagnosticBag diagnostics,
-            BoundImplicitReceiver implicitReceiver)
+            BoundLValuePlaceholder implicitReceiver)
```

Comment:
We are using the term "lvalue" here but the implicit receiver is never an lvalue, it's always an rvalue (at least from the code perspective).